EpiResponsivePicture/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -61,6 +69,36 @@ public static class ServiceCollectionExtensions
 
         return services;
     }
+    
+    /// <summary>
+    /// Creates container with provided name, when it does not exist
+    /// </summary>
+    /// <param name="options">Azure blob storage cache options</param>
+    private static void CreateContainerIfNotExists(AzureBlobStorageCacheOptions options)
+    {
+        AzureBlobStorageCache.CreateIfNotExists(options, PublicAccessType.None);
+    }
+    
+    /// <summary>
+    /// Checks if necessary options were provided, otherwise throws exception
+    /// </summary>
+    /// <param name="options">Azure blob storage cache options</param>
+    /// <exception cref="Exception">Thrown when any of required options is missing</exception>
+    private static void ValidateOptions(AzureBlobStorageCacheOptions options)
+    {
+        if(string.IsNullOrEmpty(options.ConnectionString))
+        {
+            _validationErrors = "Need to provide connection string!";
```

Comment:
I think it's clearer not to concatenate error and throw them in single exception but rather (simplified). Also, it's better to use more specific exception than generic `Exception`. Maybe `ArgumentException`?:

if (connectionString)
{
throw new ArgumentException(message)
}
if (containerName)
{
throw new ArgumenException(containerMessage)
}